### PR TITLE
AYR-671 - Record Arrangement

### PR DESCRIPTION
**Pull request by colinbowen**
## Changes in this PR
- add styles for record arangment
- update tests

## JIRA ticket
https://national-archives.atlassian.net/browse/AYR-671

### After
Screenshot 2024-03-12 at 16 45 28

Screenshot 2024-03-12 at 16 45 47

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

File: app/static/src/scss/includes/_record.scss [hunk 1]
```diff
@@ -209,6 +208,57 @@
...
+}
+
+.record-container .record-arrangement-list li::after {
+  content: " ";
+  display: block;
+  position: absolute;
+  width: var(--border-width);
+  left: calc((var(--ball-size) / 2) + (var(--border-width) / 2));
+  top: calc(var(--ball-size) + var(--border-width));
+  bottom: -3px; /*  Minus the --border-width (can't make this negative in CSS so have had to hard-code it)  */
```
Comment (anthonyhashemi) [inline app/static/src/scss/includes/_record.scss]: apparently `calc(var(--border-width) * -1);` should work

**Review by anthonyhashemi — DISMISSED** [review]

File: app/static/src/scss/includes/_record.scss (hunk 1 above)
Comment (colinbowen) [reply app/static/src/scss/includes/_record.scss]: Added this change, thanks

**Comment by pgandhizaizi** [comment]
looks good, please wait until Candace PR AYR-785 get done and merged to main

**Comment by anthonyhashemi** [comment]
will reapprove after merge conflicts fixed

**Review by anthonyhashemi — APPROVED** [review]

**Pull request merged**